spec/integration/inventory_spec.rb

```diff
@@ -152,7 +152,7 @@ module Hawkular::Inventory::RSpec
         it 'Should list resources for feed' do
           resources = @client.list_resources_for_feed feed_id
 
-          expect(resources.size).to be(2)
+          expect(resources.size).to be >= 2
```

Comment:
Yes, that's for another PR.

Some days ago, I was thinking that we should manually place metrics, alerts and inventory under another tenant and don't rely on the agent. Using another tenant will make the test data "safe" from being updated because of the agent running in hawkular services.

Whatever we are trying to test here, it is related to "feeds". And feeds is a general concept in inventory, not agent specific. We should be able to manually register a feed and turn this test into a predictable one.